lib/bcdice/game_system/SwordWorld2_0.rb

```diff
@@ -184,15 +83,17 @@ def eval_game_system_specific_command(command)
           else
             growth
           end
-        when TRANSCENDENT_TEST_RE
-          m = Regexp.last_match
-
-          critical_value = m[1].to_i
-          modifier = m[2] ? ArithmeticEvaluator.eval(m[2]) : 0
-          cmp_op = m[3]
-          target = m[3] && m[4].to_i
+        when /^2D6?@(\d+)/i
+          critical_value = Regexp.last_match(1).to_i
+          transcendent_parser = Command::Parser.new(/2D6?@\d+/i, round_type: BCDice::RoundType::CEIL)
+                                               .restrict_cmp_op_to(nil, :>=, :>)
```

Comment:
```suggestion
          transcendent_parser = Command::Parser.new(/2D6?@\d+/i, round_type: BCDice::RoundType::CEIL)
                                               .enable_critical
                                               .restrict_cmp_op_to(nil, :>=, :>)
```

`@` の値は `Command::Parser#enable_critical` を有効にして `Command::Parsed#critical` から取得してください。

https://github.com/bcdice/BCDice/blob/bc32a3031f4cf90c298a6438da1b628ae88ea91a/lib/bcdice/game_system/DoubleCross.rb#L229-L231

Reply:
`@\d+` まで含めるとクリティカル値のパースができなかったので、 `/2D6?/` のみをコマンドヘッダとしてレビュー反映を行いました。